draft-ietf-httpbis-rfc6265bis.md

```diff
@@ -1250,21 +1250,24 @@ user agent MUST process the cookie-av as follows.
 If the attribute-name case-insensitively matches the string "Domain", the user
 agent MUST process the cookie-av as follows.
 
-1.  If the attribute-value is empty, the behavior is undefined. However, the
-    user agent SHOULD ignore the cookie-av entirely.
+1.  If the attribute-value is empty:
 
-2.  If the first character of the attribute-value string is %x2E ("."):
+    1. Let cookie-domain be the canonicalized request-host.
 
-    1.  Let cookie-domain be the attribute-value without the leading %x2E
+    Otherwise:
+
+    2.  If the first character of the attribute-value string is %x2E ("."):
```

Comment:
Hmm, I get what this is trying to say, but the indentation makes it somewhat confusing. Can we unnest this somehow?

e.g.

step 0: let cookie-domain be the attribute-value.

step 1: if cookie-domain starts with `.`, let cookie-domain be cookie-domain without its leading character.

step 2: if cookie-domain is empty, let cookie-domain be the canonicalized request host.

Reply:
Agreed, we should flatten this.